suites/reef/cephfs/tier-3_cephfs_bugs.yaml

```diff
@@ -231,3 +232,9 @@ tests:
       polarion-id: CEPH-83586730
       desc: validate standby reply node not removed after setting mds_inject_health_dummy
       abort-on-fail: false
+  - test:
+      name:  Validate the details of all MDS servers are displayed with command ceph mds metadata
+      module: cephfs_bugs.validate_mds_metadata.py
+      polarion-id: CEPH-83583721
+      desc: Validate the details of all MDS servers are displayed with command ceph mds metadata
+      abort-on-fail: false
```

Comment:
Please add TC - #CEPH-83583721 test module in this file as the suite file was missing in previous PR

Reply:
Updated the suite file